### DISCUSSION TITLE
欢迎访问讨论页🔥

### DISCUSSION THREAD

**Discussion by RockChinQ** [langbot-app/LangBot#320] [Announcements]
请随意提问

**Comment by Syrzy** [comment]
视频教程被b站封了，请问还有获取的渠道吗？

**Reply by RockChinQ** [reply]
这个是新的

【【保姆级教程】手把手教你部署ChatGPT的QQ机器人，难道现在还有不会部署的吗？最好用的ChatGPT+QQ机器人-哔哩哔哩】 https://b23.tv/KnTswDz

**Reply by Syrzy** [reply]
谢谢！

**Comment by 0x676e67** [comment]
可以支持一下网页版API，使用我项目自建服务端 https://github.com/gngpp/opengpt

**Reply by RockChinQ** [reply]
感谢提供，网页版api为逆向工程，本项目仅在 revLibs 插件中支持逆向工程接口，并且已支持 acheong08/ChatGPT

**Reply by Syrzy** [reply]
感谢！

**Comment by fwerkor** [comment]
经过无数次尝试，包括使用不同的问法、不同的模型，Ai始终无法学会联网，!func检查正常。

**Reply by fwerkor** [reply]
“抱歉，由于我当前的环境无法访问互联网，我无法打开您提供的链接来查看其中的内容。然而，如果您愿意，您可以提供一些与链接相关的信息，例如链接中所涉及的主题、内容或关键细节，我会尽力为您提供相关的信息或回答问题。如果您有其他问题或需求，请随时告诉我，我会尽力提供帮助。”

**Reply by RockChinQ** [reply]
附上!func输出、使用的插件、提问的内容、控制台输出去提issue

**Reply by fwerkor** [reply]
不用了，问题解决了，谢谢

**Reply by fwerkor** [reply]
新的问题又出现了。。。已发Issue

**Comment by LiJunXiang1234** [comment]
我的api免费额度已经过期了，请问可以使用access_token完成chatgpt的接入吗？

**Reply by RockChinQ** [reply]
可以的，请查看https://github.com/RockChinQ/revLibs ，同时欢迎关注 可将逆向工程库转换成 OpenAI 标准 API 的项目 [free-one-api](https://github.com/RockChinQ/free-one-api)

**Comment by 1nio** [comment]
### 我想问一下怎么将bot返回消息里的md格式的图片链接以图片的形式显示出来

> 消息平台：LiteLoaderQQNT

例如：回复的消息中包含`![image](image.webp)`这样的图片链接，怎么让bot正确的把这个图片发出来？

**Reply by RockChinQ** [reply]
可以写个插件处理一下这个消息，转换成包含图片组件的消息链。

**Comment by osongsongo** [comment]
qq机器人的教程还能补档吗，或者单独发我一份可以吗

**Reply by RockChinQ** [reply]
可以看下我们的文档哈 https://langbot.app

**Reply by osongsongo** [reply]
qq机器人怎么配置没太看明白，要是有个完整的教程实例就好了

**Reply by the-lazy-me** [reply]
> qq机器人怎么配置没太看明白，要是有个完整的教程实例就好了

看这个 https://docs.langbot.app/insight/community.html

**Comment by 15sdwweqqe** [comment]
飞书接入langbot，使用的硅基流动的模型
一直提示无法找到模型，我应该怎么设置正确的模型名称？
[02-14 21:55:57.327] controller.py (189) - [ERROR] : 处理请求时出错 query_id=3 stage=PreProcessor : 无法确定模型 SiliconFlow/DeepSeek-R1 的信息，请在元数据中配置

**Reply by RockChinQ** [reply]
https://docs.langbot.app/config/metadata/llm-models.html
